src/plotting/check_plots.py

```diff
@@ -75,7 +87,7 @@ def check_plots(args, root_path):
         log.info(f"\tLocal sk: {plot_info.local_sk}")
         total_proofs = 0
         try:
-            for i in range(num):
+            for i in range(num_start, num):
```

Comment:
Wouldn't it make more sense to go from num_start to num_start + num?
If I do --challenge-start 1000 -num 10  I expect it to go from 1000 to 1010.

Reply:
You are right. Changed it.